float32 mean_bits
float32 voltage
uint8 pin
